visualization/include/pcl/visualization/qvtk_compatability.h

```diff
@@ -6,19 +6,45 @@
  *
  *  All rights reserved
  */
+#include <pcl/pcl_macros.h>
 #include <pcl/pcl_config.h>
 
 #if HAVE_QVTK
-  #include <vtkVersion.h>
+#include <vtkVersion.h>
 
-  #if VTK_MAJOR_VERSION > 8
-    #include <QVTKOpenGLNativeWidget.h>
-    using PCLQVTKWidget = QVTKOpenGLNativeWidget;
-  #else
-    #include <QVTKWidget.h>
-    using PCLQVTKWidget = QVTKWidget;
-  #endif
+#if VTK_MAJOR_VERSION > 8
+  #include <QVTKOpenGLNativeWidget.h>
+  using PCLQVTKWidget = QVTKOpenGLNativeWidget;
+#else
+  #include <QVTKWidget.h>
+  using PCLQVTKWidget = QVTKWidget;
+#endif
+
+
+inline auto PCL_EXPORTS getInteractorCompat(PCLQVTKWidget* qvtk) {
```

Comment:
Then  I'll have to derefence everywhere we use the methods?

Reply:
Since the function is exposed, it'll be safer to use a reference as a parameter and dereference it in the call site.

If it was not exposed, it wouldn't matter